tests/backup/backup_test.go

```diff
@@ -6522,16 +6522,18 @@ var _ = Describe("{ScheduleBackupCreationSingleNS}", func() {
 			schPolicyUid, _ = Inst().Backup.GetSchedulePolicyUid(orgID, ctx, periodicPolicyName)
 			for _, namespace := range bkpNamespaces {
 				backupName = fmt.Sprintf("%s-%s", BackupNamePrefix, namespace)
-				schBackupName, err = CreateScheduleBackup(backupName, SourceClusterName, backupLocationName, backupLocationUID, []string{namespace},
+				err = CreateSchedule(backupName, SourceClusterName, backupLocationName, backupLocationUID, []string{namespace},
 					labelSelectors, orgID, "", "", "", "", periodicPolicyName, schPolicyUid, ctx)
-				dash.VerifyFatal(err, nil, fmt.Sprintf("Verification of creating schedule backup - %s", schBackupName))
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Verification of creating [%s] schedule", backupName))
```

Comment:
Any specific reason of removing the word "backup"?

Reply:
I overlooked the need to rename the "backupName" variable to "scheduleName" before passing it to the "CreateSchedule" function.  I will rectify it.